packages/features/ee/organizations/pages/settings/profile.tsx

```diff
@@ -33,6 +33,8 @@ import {
   SkeletonText,
   TextField,
 } from "@calcom/ui";
+// if I include this in the above barrel import, I get a runtime error that the component is not exported.
+import { OrgBanner } from "@calcom/ui";
```

Comment:
As code comment says, I was getting runtime errors if I didn't import this separately.